docs/examples.rst

```diff
@@ -336,8 +336,8 @@ Fetch various data about the current state of the ERC20 contract.
     '🦄'
     >>> erc20.functions.decimals().call()
     0
-    >>> erc20.functions.totalSupply().call()
-    2040
+    >>> total_supply = assert erc20.functions.totalSupply().call()
+    >>> assert total_supply > 2040
```

Comment:
Ah, brilliant! I think this should be `>=` though? And I think there's an extra `assert` on line 339?

Reply:
It's 2044 now I think so `>` should be fine.

Reply:
Ha just saw that :) Looks good to me!